### DISCUSSION TITLE
the enviroment vars have stopped working in 9.4.1-canary.5

### DISCUSSION THREAD

**Discussion by sambhav-gore** [vercel/next.js#12936] [Help] [answered]
well It was fine in 9.4.0 I changed to 9.4.1-canary.5 to get the hot reloading solved, while hot reload is fine now the env vars have stopped working :-(

**Comment by ijjk** [comment, marked as answer]
Hi, can you provide a repo with a reproduction for this? We have tests covering the environment variables support [here](https://github.com/zeit/next.js/blob/canary/test/integration/env-config/test/index.test.js) so if something broke between those versions it is most likely due to something specific within your project.

**Reply by sambhav-gore** [reply]
well, It is not a problem on next side, I was setting env variables on the container, but I am realizing that I need to add it on my ci as they are build time
